### DISCUSSION TITLE
Button mapping

### DISCUSSION THREAD

**Discussion by gabrielmll** [dokutan/mouse_m908#37] [Q&A]
Hello all,

first of all, thank you very much for developing this great tool.
I have an m908 to use on Ubuntu 20.04.

I wish to setup a few actions, but I can't find what's the right string for them:
- volume up (tried volume_up so far)
- volume down (tried volume_down)
- cut (tried cut)
- copy (tried copy)
- paste (tried paste)

Those didn't work, and after getting the current mouse settup, the file says:
"unknow, please report as bug: 90 0 0 0"

![image](https://user-images.githubusercontent.com/2589787/151176834-b9e0fd14-5c4a-4797-9bed-8c9fb0f679e1.png)

Thank you very much

**Comment by dokutan** [comment]
``Media_Volume_Up`` and ``Media_Volume_Down`` ``Cut``, ``Copy`` and ``Paste`` should work, just make sure these buttons are actually handled by Ubuntu. The whole list is availabe here: https://github.com/dokutan/mouse_m908/blob/master/keymap.md
